modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -91,9 +90,7 @@ object ObservationModel extends ObservationOptics {
     observationId: Observation.Id,
     existence:     Input[Existence]   = Input.ignore,
     name:          Input[String]      = Input.ignore,
-    status:        Input[ObsStatus]   = Input.ignore,
-    asterismId:    Input[Asterism.Id] = Input.ignore,
-    targetId:      Input[Target.Id]   = Input.ignore
```

Comment:
I'll remove the asterism and target id from the normal observation editor so that we don't have to support two different ways of doing this.  When you link a subject to an observation it has to generate extra events so it is best not to try to do it here.

Reply:
Doesn't allowing specifying a `subject` during ObservationCreation have the same issue? I think an edit event should be sent for that subject and it isn't right now.

Reply:
Yes, that's a good point.  I updated to send target/asterism events as appropriate on observation creation as well.  It's kind of convoluted code though :-/

Reply:
Would it be better/easier to just not allow the subject to be set during creation? I don't think that would be an issue for Explore. Do you think it would be an issue for API users?

Reply:
That was my first instinct and I debated it.  It is certainly easier and more consistent with general updating which doesn't allow it.  It would require two API calls to create an observation with a target instead of just one though and it seemed like a common use case would be to load a target list and then create observations for individual targets.

I don't know, I'm certainly willing to back it out.

Reply:
Explore, at the moment at least, treats them as separate steps. I don't have a good idea of how people who work directly with the UI will be using it.

Reply:
It might not have made the doc, but we anticipated selecting a target and clicking the +Obs button to get a popup like this one:

![Screen Shot 2021-03-08 at 14 52 21](https://user-images.githubusercontent.com/4906023/110360956-24d14200-801e-11eb-9564-6828da2b422d.png)

which implies that it all happens in a transaction.  There would be similar pages in the constraints and configuration pages presumably.

Reply:
Perhaps it is in the docs, but not yet in Explore.

Reply:
If we include the subject in Create, it seems like we should also include it in Edit. And maybe we wouldn't need a separate Set Subject? Admittedly, if we included Subject, Contraints and Configuration in the Edit, the code for sending notifications would get complicated. So, I guess if we expect subsequent setting of subject/constraint/config to be done individually, having SetSubject, SetContraintSet and SetConfiguration is OK.

Reply:
> If we include the subject in Create, it seems like we should also include it in Edit.

I believe it should be an option to set up these links during observation creation because it needs to be done as a single transaction to accommodate interactions like the "New" dialog I posted.  Once it is available there, I agree putting it in the editor as well seems natural, though I do think that thereafter you are editing the links individually.

The problem is it may be a bit challenging.  I'll give it a shot though.

Reply:
The latest version again allows asterism/target in the editor.